HPXMLtoOpenStudio/resources/battery.rb

```diff
@@ -0,0 +1,80 @@
+# frozen_string_literal: true
+
+class Battery
+  def self.apply(model, battery)
+    obj_name = battery.id
+
+    default_values = Battery.get_battery_default_values()
+
+    voltage = battery.voltage # V
+    capacity = get_kWh_from_Ah(battery.capacity, voltage) # kWh
+
+    return if capacity <= 0 || voltage <= 0
+
+    is_outside = (battery.location == HPXML::LocationOutside) || (battery.location == HPXML::LocationExteriorWall) || (battery.location == HPXML::LocationRoofDeck)
+    if not is_outside
+      frac_sens = 1.0
+    else # Internal gains outside unit
+      frac_sens = 0.0
+    end
+
+    # The following calculations are from Rohit C.
+    number_of_cells_in_series = Integer((voltage / 3.6).round)
+    number_of_strings_in_parallel = Integer(((capacity * 1000.0) / (voltage * 3.2)).round)
+    battery_mass = (capacity / default_values[:capacity]) * 99.0 # kg
```

Comment:
I would think we should hardcode the denominator here, so that the assumed battery mass doesn't change should we decide to update the default battery capacity.